config/database.php

```diff
@@ -49,7 +48,6 @@
 			'database' => env('DB_DATABASE', 'forge'),
 			'username' => env('DB_USERNAME', 'forge'),
 			'password' => env('DB_PASSWORD', ''),
-			'unix_socket' => env('DB_SOCKET', ''),
```

Comment:
I haven't tested using sockets here, but if they weren't I don't see why we should block them.

Reply:
Please keep it! I am using sockets on my productive system and on my test system. I prefer sockets, because they cannot be accidentally exposed to the external world and they do not have the overhead of a TCP connection.

Reply:
It is used here: https://github.com/doctrine/dbal/blob/e15d05d08172ceeb4f55066feed48d688fdb0250/src/Driver/Mysqli/Driver.php#L60-L68